test/integration/features/step_definitions/github-api-steps.js

```diff
@@ -10,18 +10,20 @@ const user = any.simpleObject();
 Before(function () {
   nock.disableNetConnect();
 
-  githubScope = nock('https://api.github.com/');
+  // githubScope = nock('https://api.github.com/');
 });
 
 After(() => {
   nock.enableNetConnect();
   nock.cleanAll();
 
-  assert.isTrue(githubScope.isDone());
+  assert.isTrue(githubScope?.isDone() ?? true);
 });
 
-Given('a personal access token is defined for api.github.com', async function () {
+Given('a personal access token is defined for {}', async function (domain) {
```

Comment:
```suggestion
Given('a personal access token is defined for {string}', async function (domain) {
```

Reply:
`{string}` doesn't work. [Defined](https://github.com/cucumber/cucumber-expressions#parameter-types) as:
> Matches single-quoted or double-quoted strings, for example "banana split" or 'banana split' (but not banana split). Only the text between the quotes will be extracted. The quotes themselves are discarded. Empty pairs of quotes are valid and will be matched and passed to step code as empty strings.

While `{}` is a regex to match anything (`/.*/`).

Reply:
interesting. i guess i never looked deep enough to understand that distinction. i think i normally just go ahead and add the quotes to the feature file to make the typed version work. have thoughts against quoting the values in the feature file. i dont think i feel too strongly either way